src/test/app/AmendmentTable_test.cpp

```diff
@@ -91,19 +110,34 @@ class AmendmentTable_test final : public beast::unit_test::suite
 
     std::unique_ptr<AmendmentTable>
     makeTable(
+        Application& app,
+        std::chrono::seconds majorityTime,
+        Section const supported,
+        Section const enabled,
+        Section const vetoed)
+    {
+        auto t = make_AmendmentTable(
+            app, majorityTime, supported, enabled, vetoed, journal);
+        return t;
+    }
+
+    std::unique_ptr<AmendmentTable>
+    makeTable(
+        test::jtx::Env& env,
         std::chrono::seconds majorityTime,
         Section const supported,
         Section const enabled,
         Section const vetoed)
     {
         return make_AmendmentTable(
-            majorityTime, supported, enabled, vetoed, journal);
+            env.app(), majorityTime, supported, enabled, vetoed, journal);
     }
```

Comment:
The other override of `makeTable` calls the base `makeTable`. This could, too.
```
        return makeTable(
            env.app(), majorityTime, supported, enabled, vetoed, journal);
```

Reply:
Done.